tracer/src/Datadog.Trace/Configuration/ConfigurationSources/Telemetry/OpenTelemetryHelpers.cs

```diff
@@ -0,0 +1,72 @@
+// <copyright file="OpenTelemetryHelpers.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.Telemetry.Metrics;
+
+namespace Datadog.Trace.Configuration.ConfigurationSources.Telemetry
+{
+    internal static class OpenTelemetryHelpers
+    {
+        public static void GetConfigurationMetricTags(
+                string openTelemetryKey,
+                out MetricTags.OpenTelemetryConfiguration openTelemetryConfig,
+                out MetricTags.DatadogConfiguration datadogConfig)
+        {
+            if (string.Equals(openTelemetryKey, "OTEL_SERVICE_NAME", StringComparison.OrdinalIgnoreCase))
```

Comment:
You will likely have to (or maybe _should_ add these to the tests we have to make sure these are recorded in config_norm.js)